tests/functional/replication/probe.spec.js

```diff
@@ -14,7 +14,7 @@ function mockQueueProcessor() {
     };
 }
 
-describe('Probe server', () => {
+describe.only('Probe server', () => {
```

Comment:
why `only`?

Reply:
grrr sorry, really need to enable the eslint that checks that...